docs/Using Price Feeds/l2-sequencer-feeds/l2-sequencer-flag.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: "L2 Sequencer Uptime Feeds"
+permalink: "docs/l2-sequencer-flag/"
+---
+
+Optimistic rollup protocols move all execution off the layer 1 (L1) Ethereum chain, complete execution on a layer 2 (L2) chain, and return the results of the L2 execution back to the L1. These protocols have a [sequencer](https://community.optimism.io/docs/how-optimism-works/#block-production) that executes and rolls up the L2 transactions by batching multiple transactions into a single transaction.
+
+If a sequencer becomes unavailable, it is impossible to access read/write APIs that consumers are using and applications on the L2 network will be down for most users without interacting directly through the L1 optimistic rollup contracts. The L2 has not stopped, but it would be unfair to continue providing service on your applications when only a few users can use them.
+
+To help your applications identify when the sequencer is unavailable, you can use a data feed that tracks the last known status of the sequencer at a given point in time. This is to allow customers to prevent mass liquidations by providing a grace period to allow customers to react to such an event.
+
+Because not all L2 networks are architected the same way, the Arbitrum sequencer feed behaves slightly differently than the Optimism and Metis sequencer feeds. The architecture and process for handling outages is different. Follow the instructions for your specific network; either [Arbitrum](#arbitrum) or [Optimism and Metis](#optimism-and-metis).
+
+**Table of Contents**
+
+- [Arbitrum](#arbitrum)
+  - [Architecture on Arbitrum](#architecture-on-arbitrum)
+  - [Handling Arbitrum outages](#handling-arbitrum-outages)
+  - [Example code for Arbitrum](#example-code-for-arbitrum)
+- [Optimism and Metis](#optimism-and-metis)
+  - [Architecture on Optimism and Metis](#architecture-on-optimism-and-metis)
+  - [Handling outages on Optimism and Metis](#handling-outages-on-optimism-and-metis)
+  - [Example code for Optimism and Metis](#example-code-for-optimism-and-metis)
+
+## Arbitrum
+
+Although the proxy contracts are similar, the L2 sequencer uptime feed on Arbitrum behaves differently than the feeds on [Optimism and Metis](#optimism-and-metis).
```

Comment:
I think you can remove this for now as well.

Reply:
Removed